### ISSUE TITLE
Availability of pseudo-label datasets for layerwise/progressive distillation

### ISSUE THREAD

**Issue by full-powers** [jxiw/MambaInLlama#32]
Hello,

I am trying to reproduce the Llama-3-8B layerwise/progressive distillation using:

- `mamba_llama/llama3_0.25_mamba.yaml`
- `mamba_llama/llama3_0.50_mamba.yaml`
- `train_mamba/train_hybrid.py`

The configs contain the following placeholders:

```yaml
train_datasets_path: [PSEUDO_LABEL_PATH_1, PSEUDO_LABEL_PATH_2]
```

The training code expects input_ids.pt and labels.pt under each dataset directory.
However, I could not obtain the exact UltraChat/UltraFeedback pseudo-label files used for the paper experiments.
The download link in the README is either unavailable to me or does not provide enough information to identify the files corresponding to the Llama-3-8B experiments.

As mentinoed in your README:

> Generate pseudo labels from a teacher model meta-llama/Meta-Llama-3-8B-Instruct. We provide the generated pseudo labels using the seed dataset of the UltraChat and UltraFeedback dataset [here](https://drive.google.com/drive/folders/1KzmFOJ6_pBZOuSYQKDj5jSD5rsvzFy-U?usp=sharing). Please download it and change the train_datasets_path in llama3_0.25_mamba.yaml and llama3_0.50_mamba.yaml to the path of your downloaded llama3_ultrafeedback and llama3_ultrachat.

Could you please let me know:
Whether the original pseudo-label files are still available;
Which files should be used for the Llama-3-8B 25% and 50% Mamba stages;
Whether the same files are reused across both stages; and
Whether the pseudo-label generation code and decoding settings are available.
I am specifically trying to reproduce the Mamba-Llama3 (50%) results in Table 3.
Thank you!
